src/software/jetson_nano/primitive_executor.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
+#include "proto/primitive.pb.h"
+#include "software/geom/vector.h"
+#include "software/world/world.h"
+
+class PrimitiveExecutor
+{
+   public:
+    /**
+     * Start running a primitive
+     *
+     * NOTE: This function is not thread-safe.
```

Comment:
why this comment?

Reply:
Historically the primitive manager has been thread safe so I thought I'd add a comment

but removed it because we label things as thread safe, unlabeled just means not thread safe